4_advanced_usage/50_script_in_directives.txt

```diff
@@ -116,10 +116,12 @@ at each policy generation.
 
 ====
 
-AIX password generation depends on the availability of *PBKDF2WithHmac256* and
-*PBKDF2WithHmac512* in the JVM. Some JVMs don't include these libraries. In
-this case, the hashing algorithm falls back to *SHA1*, and an error message
-will be logged.
+AIX password generation depends on the availability of *PBKDF2WithHmacSHA256* and
+*PBKDF2WithHmacSHA512* in the JVM. These algorithms are included by default on 
+HotSpot 1.8 and OpenJDK 1.8 and upward. In the case where your JVM does not support theses
+algorithms, typically on a JVM JDK or JVM 1.7 version, the hashing algorithm falls back to 
```

Comment:
I don't understand the bit "JVM JDK or JVM 1.7". In my understanding, "JVM *" is a subset of "JVM", so why repeat this?